src/ErrorProne.NET/ErrorProne.NET/Common/RuleIds.cs

```diff
@@ -41,5 +40,10 @@ public static class RuleIds
 
         // Errors in DebuggerDisplayAttribute
         public const string DebuggerDisplayAttributeInvalidFormat = "ERP103";
+
```

Comment:
Because those contants has lower value than previous, maybe it would be reasonable to move them up to have them in order?